kedro/framework/project/__init__.py

```diff
@@ -54,8 +54,14 @@ def validator_func(settings, validators):
 class _IsSubclassValidator(Validator):
     """A validator to check if the supplied setting value is a subclass of the default class"""
 
-    def _validate_items(self, settings, env=None):
-        super()._validate_items(settings, env)
+    def _validate_items(
+        self,
+        settings: Any,
+        env: Optional[str] = None,
+        only: Optional[Union[str, Sequence]] = None,
+        exclude: Optional[Union[str, Sequence]] = None,
+    ) -> None:
+        super()._validate_items(settings, env, only, exclude)
```

Comment:
Do you think we should just be doing `**kwargs` in case dynaconf change the signature of this again in the future?

Reply:
Ideally we wouldn't be overwriting a protected method here at all I guess, but Lim said that was how they recommended doing this... Would be interesting to know what their recommendation is now given that they changed this signature 😬

Reply:
Yeah, I'm ambivalent. I considered doing `**kwargs` too, happy to change it to that if you all agree that's better. :)

Reply:
Wait, actually surely this won't work with `dynaconf<3.1.6`? Since `_validate_items` will be expecting 2 arguments and you've provided it with 4. So unless we change version requirement to `dynaconf>=3.1.6` we'll need to use `**kwargs`?

Reply:
In fact, given that we have `**kwargs` shouldn't we also include `*args` for maximum future proof-ness?

Reply:
> In fact, given that we now have `**kwargs` shouldn't we also include `*args` for maximum future proof-ness?

I'm not sure about that; passing along required, positionally arguments is a bit different from passing on these new, parameterized things with defaults IMO.